managing-os/templates/fedora/upgrade-28-to-29.md

```diff
@@ -0,0 +1,390 @@
+---
+
+Upgrading the Fedora 28 Template to Fedora 29
+=============================================
+
+This page provides instructions for performing an in-place upgrade of an
+installed Fedora 28 [TemplateVM] to Fedora 29. If you wish to install a new,
+unmodified Fedora 29 template instead of upgrading a template that is already
+installed in your system, please see the [Fedora TemplateVM] page instead.
+
+These instructions can also be used to upgrade a Fedora 26 TemplateVM to
+Fedora 29. Simply start by cloning `fedora-26` instead of `fedora-28` in the
+instructions below.
+
+
+Important information regarding RPM Fusion repos
+------------------------------------------------
+
+If your RPM Fusion repositories are **disabled** when you upgrade a TemplateVM from Fedora 28 to 29, all RPM Fusion packages and RPM Fusion repo definitions will be removed from that TemplateVM.
+If your RPM Fusion repositories are **enabled** when upgrading, all RPM Fusion packages and repo definitions will be retained and updated as expected.
+For most users, this behavior should not cause a problem, since a TemplateVM in which the RPM Fusion repos are disabled is probably a TemplateVM in which you never wish to use them.
+However, if you wish to have the RPM Fusion repo definitions after upgrading in a TemplateVM in which they are currently disabled, you may wish to temporarily enable them prior to upgrading or manually create, copy, or download them after upgrading.
+
+
+Workaround for `python2-xcffib` upgrade error
+---------------------------------------------
+
+When attempting to upgrade from Fedora 26 or 28 to Fedora 29, you may encounter an error similar to this:
+
+    Error: Transaction check error:
+      file /usr/lib/python2.7/site-packages/xcffib-0.5.1-py2.7.egg-info/PKG-INFO from install of python2-xcffib-0.5.1-5.fc29.noarch conflicts with file from package python-xcffib-0.5.1-1.fc26.noarch
+      file /usr/lib/python2.7/site-packages/xcffib/_ffi.pyc from install of python2-xcffib-0.5.1-5.fc29.noarch conflicts with file from package python-xcffib-0.5.1-1.fc26.noarch
+      file /usr/lib/python2.7/site-packages/xcffib/_ffi.pyo from install of python2-xcffib-0.5.1-5.fc29.noarch conflicts with file from package python-xcffib-0.5.1-1.fc26.noarch
+      file /usr/lib/python2.7/site-packages/xcffib/xinput.pyc from install of python2-xcffib-0.5.1-5.fc29.noarch conflicts with file from package python-xcffib-0.5.1-1.fc26.noarch
+      file /usr/lib/python2.7/site-packages/xcffib/xinput.pyo from install of python2-xcffib-0.5.1-5.fc29.noarch conflicts with file from package python-xcffib-0.5.1-1.fc26.noarch
+
+To work around this error:
+
+1. Upgrade while excluding the problematic packages by using `-x python2-xcffib -x qubes-gui-vm -x qubes-gui-agent`.
+2. Upgrade `python2-xcffib` using `sudo dnf swap python-xcffib python2-xcffib`.
+   (This should automatically upgrade the other excluded packages too.)
+
+
+Qubes 3.2 Instructions
```

Comment:
There is no Fedora 29 support for Qubes 3.2, so this whole section should be removed. And maybe even replaced with such statement?